src/pickle_filter.js

```diff
@@ -47,7 +48,7 @@ export default class PickleFilter {
   }
 
   matchesAnyLine({ pickle, uri }) {
-    const lines = this.featureUriToLinesMapping[uri]
+    const lines = this.featureUriToLinesMapping[path.resolve(uri || '')]
```

Comment:
Does a test fail if the `|| ''` is removed? I don't understand why that is needed

Reply:
`resolve` requires a string argument, could rewrite as:

```javascript
const lines = this.featureUriToLinesMapping[uri ? path.resolve(uri) : undefined]
```
or just return true if uri is undefined, which is the case when running with a line that does not exist

Reply:
I can't think of a case where this function would be called without a `uri` though.

Reply:
`uri` is undefined during all of the `line, name, and tag filters` tests

Reply:
Auhh. Can you please update the unit tests so a uri is always passed in? I'd prefer not to have the `|| ''` since in real use I expect the uri to always be there